### ISSUE TITLE
Entities applicable to derivatives only

### ISSUE THREAD

**Issue by Lestropie** [bids-standard/bids-specification#1528]
In stalking the Copenhagen BIDS Derivatives meeting, I passed comment on the fact that as the scope of BIDS Derivatives increases, there will be an increasing number of entities that are not considered valid for raw BIDS datasets, and are valid only for BIDS Derivative datasets, and that a clear distinction regarding this will need to be established in the specification. 

@Remi-Gau suggested that there already exists some level of distinction. I searched and found that for some entities, eg. [`atlas`](https://github.com/bids-standard/bids-specification/blob/v1.8.0/src/schema/objects/entities.yaml#L35), the string "This entity is only applicable to derivative data" appears in the Description.

What I propose is that in time a stronger distinction will be necessary.

Some thoughts (certainly not an exhaustive list, happy to seed a more general discussion):

-   In Appendix -> Entities, those applicable to only derivative data should appear as a secondary list, rather than being interspersed with those applicable to raw data.

-   In Appendix -> Entity Table, both entities and suffices applicable to only derivative data appear to be absent, but this is not stated at the head of the page. If this is considered to be beyond scope of the specification, this may need to be stated explicitly. Alternatively, as per the first point, a second table that incorporates both raw and derivative entities and suffices could be defined, with the current table preserved as-is (in order to not complicate interpretation for raw data).

-   Both points above necessitate intrinsic binary flags for both entities and suffices indicating whether that item applies exclusively to derivative data, which would then be used internally to allocate those items to tables dedicated to derivative entries.

**Comment by Remi-Gau** [comment]
I was actually more referring to the fact that some entities only appear in the schema for [derivatives files](https://github.com/bids-standard/bids-specification/tree/master/src/schema/rules/files/deriv), but never for [raw files](https://github.com/bids-standard/bids-specification/tree/master/src/schema/rules/files/raw).

See for example:
https://github.com/bids-standard/bids-specification/blob/f2cea0f43444b55c9885a9717b25272b27790959/src/schema/rules/files/deriv/imaging.yaml#L2-L9

I agree that the 2 additions to the appendices would be most welcome.

I suspect that whether an entity is raw or derivatives should be probably be done by modifying the schema in the following file:
- https://github.com/bids-standard/bids-specification/blob/master/src/schema/rules/entities.yaml

The sentence would `"This entity is only applicable to derivative data"` could then be removed from the definition of entities and we would let the rendering macro add it where needed.

@effigies @tsalo any thoughts?

**Comment by oesteban** [comment]
I agree the schema seems the most unambiguous place for this distinction
